files/galaxy/dynamic_job_rules/production/total_perspective_vortex/tools.yml

```diff
@@ -1020,8 +1020,8 @@ tools:
     - if: 5 <= input_size < 20
       cores: 16
     - if: input_size >= 20
-      cores: 120
-      mem: 1922
+      cores: 30
```

Comment:
This is another heavily cpu bound tool (actually all of them are..) so we should do some investigation into their scaling (on a tool by tool basis) before we make these changes. Are they linear? Are they exponential? What is the point of diminishing returns?